tools/base/src/index.ts

```diff
@@ -13,3 +13,4 @@ governing permissions and limitations under the License.
 export * from './Base.js';
 export * from './sizedMixin.js';
 export * from 'lit';
+export * from 'lit/async-directive.js';
```

Comment:
Great to see this in the repro!

Please move this to a `async-directive.ts` file that mirrors `directive.ts` so that tree shaking is less required to exclude this code from a project.

Then I think we're good to ship this!

Reply:
Done